### PR TITLE
Add 2023-07-26 agenda and minutes

### PR DESCRIPTION
**Pull request by csarven**
Request to merge in 24 hours following meeting.

### CODE REVIEW

File: meetings/2023-07-26.md [hunk 1]
```diff
@@ -28,67 +35,131 @@
...
+ACTION: SC to update. 
 
 #### Consider updating last call and approval dates
-* SC: Raised by MC that folks may be on holidays. Last call: 2023-07-25, Approval: (after) 2023-08-01 dates were proposed ( https://github.com/solid/process/pull/323#issuecomment-1629875377 ) . WT hinted at 28 days being sufficient in recent comments.
-* SC: Shall we extend for another two weeks? Let's get more reviews in.
-
-PROPOSAL: Change Last Call to 2023-08-01 and finalise by 2023-08-08.
+* SC: Raised by MC that folks may be on holidays. Last call: 2023-07-25, Approval: (after) 2023-08-01 dates were proposed ( https://github.com/solid/process/pull/323#issuecomment-1629875377 ) . This was accepted in prior calls. WT hinted at 28 days being sufficient in recent comments and extending further may not particularly help considering current state.
+* SC: Reviews seem sufficient to me but always welcome to have more representation, so getting more reviews would be great.
+* SC: Here are two proposals but let's get other feedback or alternative proposals before deciding anything.
```
Comment (TallTed) [inline meetings/2023-07-26.md:123]:
```suggestion
* SC: Raised by MC that folks may be on holidays. Last call: 2023-07-25, Approval: (after) 2023-08-01 dates were proposed (https://github.com/solid/process/pull/323#issuecomment-1629875377). This was accepted in prior calls. WT hinted at 28 days being sufficient in recent comments and extending further may not particularly help considering current state.
* SC: Reviews seem sufficient to me but always welcome to have more representation, so getting more reviews would be great.
* SC: Here are two proposals, but let's get other feedback or alternative proposals before deciding anything.
```

File: meetings/2023-07-26.md [hunk 2]
```diff
@@ -28,67 +35,131 @@
...
+
+PROPOSAL: Extend approval date (2023-08-01) to something else.
+
+PROPOSAL: Change Last Call to 2023-08-01 and Approval By 2023-08-08.
+
+* eP: We should do out best effort to address those thumbs down: Melvin and Hadrian. Melvin voiced his concerns, let's try to get Hadrian to explain. Once we clarify that we're good to go. No problem with delaying, but still. I can reach out to Hadrian to get him to comment.
+* SC: Not so much addressing the concerns/emoji. It's more about giving time to folks to review. Could the group use more time to gather more responses? If it had been 50/50 it feels yes, e should extend, but here we have more than most groups charters or even PRs having this number of approvals as well as long discussion to indicate it has been thoroughly discussed. That does not mean we need to stop it, but would another two weeks/month/year help?
```
Comment (TallTed) [inline meetings/2023-07-26.md:130]:
```suggestion
* eP: We should make our best effort to address those thumbs down: Melvin and Hadrian. Melvin voiced his concerns; let's try to get Hadrian to explain. Once we clarify that, we're good to go. No problem with delaying, but still. I can reach out to Hadrian to get him to comment.
* SC: Not so much addressing the concerns/emoji. It's more about giving time to folks to review. Could the group use more time to gather more responses? If it had been 50/50 it feels yes, we should extend, but here we have more than most group's charters or even PRs having this number of approvals as well as long discussion to indicate it has been thoroughly discussed. That does not mean we need to stop it, but would another two weeks/month/year help? 
```

File: meetings/2023-07-26.md [hunk 3]
```diff
@@ -28,67 +35,131 @@
...
+* SC: Not so much addressing the concerns/emoji. It's more about giving time to folks to review. Could the group use more time to gather more responses? If it had been 50/50 it feels yes, e should extend, but here we have more than most groups charters or even PRs having this number of approvals as well as long discussion to indicate it has been thoroughly discussed. That does not mean we need to stop it, but would another two weeks/month/year help? 
+* WT: I partially agree with eP on the fact that we could try to hear the people who reacted only through emoji, but also want to voice my concern that if they were there in the thread and had the possibility to comment but didn't feel the need to, just leaving emoji I don't feel the urge to go after them and ask them what is it that they put an emoji for. That being said, I'm in favor of extending another week or two. 
+* SC: This is not a standard for a CG, W3C has an understanding that if 5% of members approve a charter or a spec level transition it goes ahead. Not sure if it's written down somewhere. If we use that measure we are pretty much at that stage. So we have met the bar. The more reviews the better.
```
Comment (TallTed) [inline meetings/2023-07-26.md:132]:
```suggestion
* WT: I partially agree with eP on the fact that we could try to hear the people who reacted only through emoji, but also want to voice my concern that if they were there in the thread and had the possibility to comment but didn't feel the need to, just leaving emoji, I don't feel the urge to go after them and ask them what is it that they put an emoji for. That being said, I'm in favor of extending another week or two. 
* SC: This is not a standard for a CG; W3C has an understanding that if 5% of members approve a charter or a spec level transition, it goes ahead. Not sure if it's written down somewhere. If we use that measure, we are pretty much at that stage. So we have met the bar. The more reviews the better. 
```

File: meetings/2023-07-26.md [hunk 4]
```diff
@@ -28,67 +35,131 @@
...
+* SC: This is not a standard for a CG, W3C has an understanding that if 5% of members approve a charter or a spec level transition it goes ahead. Not sure if it's written down somewhere. If we use that measure we are pretty much at that stage. So we have met the bar. The more reviews the better. 
+* PAC: This is a rough understanding, but yes, for a vote to be considered relevant we expect 5% of members to vote. Not sure if we can apply to CG. My understanding is that not all member organizations are interested in all fields of work. Participants to a CG are expected to be all interested in the same topic. So not entirely comfortable to use that rule, but that being said is that participation in CG is "cheaper" than W3C.. so many participants might be inactive and not expected to give their opinion. My point is I would not invoke that rule, but that doesn't mean the 5% rule isn't reasonable≤ for totally different reasons.
```
Comment (TallTed) [inline meetings/2023-07-26.md:133]:
```suggestion
* PAC: This is a rough understanding, but yes, for a vote to be considered relevant we expect 5% of members to vote. Not sure if we can apply to CG. My understanding is that not all member organizations are interested in all fields of work. Participants to a CG are expected to be all interested in the same topic. So not entirely comfortable to use that rule, but that being said, participation in CG is "cheaper" than W3C membership.. so many participants might be inactive and not expected to give their opinion. My point is I would not invoke that rule, but that doesn't mean the 5% rule isn't reasonable≤ for totally different reasons. 
```

File: meetings/2023-07-26.md [hunk 5]
```diff
@@ -28,67 +35,131 @@
...
+* SC: PAC, do you think the amount of feedback, discussion, reviews, etc. looks healthy? The group has 272 people, compared to a WG charter or other CG charters, does it look good? Related to comment by Ted: (URL)
+* PAC: It looks healthy enough for me. It's hard to measure how many of the registered participants in the CG are still active so I guess this is the best measure we have of who is engaged at the moment and that's also a pragmatic way of dealing with that. If someone is strongly opposed to this charter and engaged enough to try and be heard, they would have already. 
+* WT: I did a count of active member over the past year on Matrix/Gitter, and there were about 50 people active. That would make the 13 respondents about a quarter. I have been waiting a few weeks in order to see if others ask for changes. Maybe we can call it as is now and if there are suggestions for changes we can always make them later.
```
Comment (TallTed) [inline meetings/2023-07-26.md:136]:
```suggestion
* WT: I did a count of active members over the past year on Matrix/Gitter, and there were about 50 people active. That would make the 13 respondents about a quarter. I have been waiting a few weeks in order to see if others ask for changes. Maybe we can call it as is now, and if there are suggestions for changes, we can always make them later. 
```

File: meetings/2023-07-26.md [hunk 6]
```diff
@@ -28,67 +35,131 @@
...
-* SC: So, at least two separate cases to communicate/demo: 1) general CG work, and 2) specific use cases or interests.
-* SC: For the time being, I suggest everyone self-publishes their screencasts, and share the URLs with the CG. We can find a home to refer to those links. More details to follow: make sure the URL is publicly archivable by the Internet Archive for instance.
+* SC: There is a "Call for Group updates and Technical Demos (videos)" at TPAC2023. It is suggested that we can showcase the work done in the Solid CG by creating a short video to be published on the TPAC2023 website. Videos are due by 2023-08-30. More details to follow but there is https://www.w3.org/wiki/TPAC/2023/Demos_and_Group_updates#Best_Practices_for_Recording_Videos .
+* SC: We have two separate cases to communicate/demo: 1) general CG work, and 2) specific use cases or interests.
+* SC: For the time being, I suggest everyone self-publishes their screencasts, and share the URLs with the CG. These are good assets. We can find a home to refer to those links. More details to follow: make sure the URL is publicly archivable by the Internet Archive for instance.
```
Comment (TallTed) [inline meetings/2023-07-26.md:162]:
```suggestion
* SC: For the time being, I suggest everyone self-publish their screencasts, and share the URLs with the CG. These are good assets. We can find a home to refer to those links. More details to follow; for instance, make sure the URL is publicly archivable by the Internet Archive.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
the usual

File: meetings/2023-07-26.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+* SC: This is not a standard for a CG, W3C has an understanding that if 5% of members approve a charter or a spec level transition it goes ahead. Not sure if it's written down somewhere. If we use that measure we are pretty much at that stage. So we have met the bar. The more reviews the better. 
+* PAC: This is a rough understanding, but yes, for a vote to be considered relevant we expect 5% of members to vote. Not sure if we can apply to CG. My understanding is that not all member organizations are interested in all fields of work. Participants to a CG are expected to be all interested in the same topic. So not entirely comfortable to use that rule, but that being said is that participation in CG is "cheaper" than W3C.. so many participants might be inactive and not expected to give their opinion. My point is I would not invoke that rule, but that doesn't mean the 5% rule isn't reasonable≤ for totally different reasons. 
+* SC: PAC, do you think the amount of feedback, discussion, reviews, etc. looks healthy? The group has 272 people, compared to a WG charter or other CG charters, does it look good? Related to comment by Ted: (URL)
```
Comment (VirginiaBalseiro) [inline meetings/2023-07-26.md:134]:
```suggestion
* SC: PAC, do you think the amount of feedback, discussion, reviews, etc. looks healthy? The group has 272 people, compared to a WG charter or other CG charters, does it look good? Related to [comment by Ted](https://github.com/solid/process/pull/323#pullrequestreview-1547868732)
```

File: meetings/2023-07-26.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+* SC: Can others review the questions? Is this generally covered by the Scope? Should it be explicitly listed?
+* eP: In terms of what will be published by CG, would that be a note? 
+* SC: My interpretation would be vocab and possibly shapes. Solid CG is the only working on it, SolidOS is interested in it.. so we should acknowledge. 
+* WT: I think that everything that will be done on UI level should/could rely on the work done in application interoperability and the panes and shapes are important but should be gathered under the work item of interop. 
+* SC: We have data shapes as an item under scope. Should we revise that item to emphasize UI? 
+* WT: Could change wording to something more general, maybe mention relationship between data/display thereof. 
+* eP: I think it doesn't seem to be out of scope. New work item describes workflow describes how new working items can be proposed for the CG. I think this should just go through this workflow.
+* SC: We don't say vocabularies but we could mention alongside data shapes.
+* WT: I think that it's in scope but need to mention specifically shapes/vocab when the point is interoperability. Maybe mentioning as examples.
```
Comment (woutermont) [inline meetings/2023-07-26.md:108]: The way this was recorded seemed to be interpretable as exactly the opposite of what I meant.

```suggestion
* WT: I think that it's certainly in scope but only mentioning shapes/vocab is too specific when the point is interoperability. Maybe mention those as examples.
```

**Review by woutermont — CHANGES_REQUESTED** [review]

**Pull request merged**